homeassistant/components/media_player/firetv.py

```diff
@@ -37,6 +37,16 @@
 DEFAULT_ADB_SERVER_PORT = 5037
 DEFAULT_GET_SOURCES = True
 
+SERVICE_ADB_CMD = 'firetv_adb_cmd'
```

Comment:
Call it `SERVICE_ADB_COMMAND` and `'firetv_adb_command'`.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/21419/commits/546d1d1b05b7f0bf1c9e9ff0490ccc374a6f2336